packages/sdk/src/modules/base/index.ts

```diff
@@ -90,17 +100,16 @@ export class Base {
 
   protected async _transactionError(
```

Comment:
Never called _transactionError in previous base/index.ts file. Now it gets called if a failed extrinsic event is found.